src/noframe.js

```diff
@@ -0,0 +1,40 @@
+export default function noframe(target, container) {
+  let els = typeof target === 'string' ? document.querySelectorAll(target) : target;
+  if (!('length' in els)) {
+    els = [els];
+  }
+  for (let i = 0; i < els.length; i++) {
+    const el = els[i];
+    const height = el.offsetHeight;
+    const width = el.offsetWidth;
+    if (typeof container !== 'undefined' && container !== 'false') {
+      const parent = document.querySelectorAll(container);
```

Comment:
`querySelectorAll` will give you an array JSYK. Not sure if you want a single element or an array. If you want a single element, just use `querySelector`